src/compilers/if.cr

```diff
@@ -25,7 +25,25 @@ module Mint
 
     def _compile(node : Ast::If, block : Proc(String, String)? = nil) : String
       condition =
-        compile node.condition
+        case statement = node.condition
+        when Ast::Statement
+          x =
```

Comment:
Don't be lazy, please :P

```suggestion
          expression =
```

Reply:
Sorry, I created the PR and then converted to draft :joy: